### ISSUE TITLE
LBRS Urok boss cant be resummoned after wipe/despawn

### ISSUE THREAD

**Issue by darthmagikal** [azerothcore/azerothcore-wotlk#8712]
### Current Behaviour

With a group of seasoned adventurers we attempted the fight "Urok Doomhowl" on Lower Blackrock Spire.
First issue is after wiping we notice the thrash ( ogres ) despawned and the boss appeared and stayed for a couple seconds. Wich mean a hunter or rogue or even alliance Night elve ,could summon the boss Vanish,Feight Death , Shadowmelt. and ignore trash and just kill boss later .
The second issue is when the boss despawned , it cant be resummon! The animation the pike and head become animated , but the boss or trash does not reappear!

### Expected Blizzlike Behaviour

https://youtu.be/W7YdNcfEDgo
Video above.
The trash ogres should spawn and remain there until killed.
The boss can be resummoned opon clicking the pile of skull

### Source

https://classic.wowhead.com/npc=10584/urok-doomhowl#comments

https://youtu.be/W7YdNcfEDgo

### Steps to reproduce the problem

1Gather the item to summon boss, combine the head and pike , click the pile
2Wait for ogres and boss.
3Die

### AC rev. hash/commit

AzerothCore rev. d3bbaa918368 2021-10-18 00:41:06 +0300 (molten_core_v3 branch) (Win64, RelWithDebInfo, Static) (worldserver-daemon)

### Operating system

windows 10

**Comment by Shineslippers** [comment]
Pike with head should disapear i guess after Urok summon. After this you obtain items again and resummon.
https://youtu.be/2RqoKtMHavk?t=2692

**Comment by patou01** [comment]
afaik this should be fixed, is it not? It's part of the PR I made :( I just don't see how you should get a second head in the same run, since it's from a boss mob. But sure, someone could bring a spare.

The trap works on my side, though my DB is a mess, I haven't touched that. Could you double check please? https://drive.google.com/file/d/1TdCDoBamspCt3MIO0pYFOs-zvMiwSsCY/view?usp=sharing (once the video is done uploading/converting)

**Comment by Shineslippers** [comment]
I'm sorry my bad, Damage is working. back to head and pike. In half of videos players skipping ths boss. There is always alot of wipes. So some people can have head from other run i guess.

**Comment by patou01** [comment]
That should work as well, it respawns the pile of skulls, and then you can get a new pike and head to restart it. I didn't know if it was needed but figured it wouldn't hurt. Is htat not working on master?

**Comment by Shineslippers** [comment]
Gimme a moment. I found something controversial. Maybe i'm wrong there.

**Comment by patou01** [comment]
Huh? The mobs don't attack the pike afaik, since I wrote the script, I'd be surprised if they did. I couldn't figure out how to do it. So the only reasonable thing after that was to make everybody despawn (or you'd get a hack to just have a rogue start the event and wait for the mobs to despawn).

I think at this stage we should close this one and the other issue regarding this boss (https://github.com/azerothcore/azerothcore-wotlk/issues/7364) and make a new issue that just points out:
1) The mobs don't attack the pike
2) The boss should only despawn if the pike is destroyed

**Comment by Shineslippers** [comment]
That's not specified how they are destroying it.Right now it's destroying with NPCs despawn what's also good way to handle it.  Only issue with Urok despawn.

**Comment by patou01** [comment]
The exploit mentioned here should be fixed now. 
The boss can be resummoned currently with a 2nd head, but the despawn behaviour is apparently not correct.

We track this remaining part in  https://github.com/azerothcore/azerothcore-wotlk/issues/9023
